.circleci/config.yml

```diff
@@ -12,13 +12,15 @@ jobs:
 
   checkESLint:
     docker:
-      - image: node:slim
+      - image: node:18
     resource_class: large      
     steps:
       - checkout
       - run:
+          environment:
+            YARN_ENABLE_IMMUTABLE_INSTALLS: false
           command: |
-            yarn; yarn lint
+            yarn set version 3.2.2 && yarn install && yarn workspace @1kv/core build && yarn workspace @1kv/core lint
```

Comment:
Right now CI will just build and lint `core` for this, but this should be changed to just `yarn build && yarn lint` in the future. The workspace scripts currently will do this for each individual package.